docs/decisions/0013-custom-groups-for-enterprise-learners.rst

```diff
@@ -0,0 +1,98 @@
+0013 Creating Custom Groups for Enterprise Customers
+##############################
+
+Status
+******
+
+**Pending**
+
+Terminology
+*******
+*Enterprise Group* - Collection of enterprise customer users connected to a policy/subsidy
+
+*Membership* - Relationship between enterprise learner and enterprise group
+
+
+Context
+*******
+
+Groups is a new feature that will allow admins the flexibility to control and distribute funding. These changes will improve the user management and budgeting from the admin perspective, as well as adding personalization to the learner experience. Groups will be associated with a learner credit budget. In the past, we’ve used workarounds to sidestep this functionality, but with this new implementation, we will increase scalability and personalization.
+
```

Comment:
if we wanted, we can maybe add a section here on how systems will interact, part of the intention for this table will be to add further filtering on the `/enterprise-learner/` endpoint - this could also maybe go into a consequences section?